dev-libs/libratbag/libratbag-0.13.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
Please also test it with Python 3.8.

Reply:
Unfortunately dev-python/python-evdev[python_targets_python3_8] isn't available.  If there is a way to work around this, let me know.

Reply:
Ok, I added support for python 3.8 and tested.  Everything appears to work fine with all 3 ebuilds.